deps/Makefile

```diff
@@ -272,6 +276,7 @@ include $(SRCDIR)/patchelf.mk
 include $(SRCDIR)/mbedtls.mk
 include $(SRCDIR)/libssh2.mk
 include $(SRCDIR)/libgit2.mk
+include $(SRCDIR)/curl.mk
```

Comment:
should be done before libgit2, and libgit2 will need to declare a dependency on it